src/controllers/redeploy.go

```diff
@@ -0,0 +1,8 @@
+package controllers
+
+func redeploy(callbackID string, data map[string]interface{}) {
+	err := stop(callbackID, data)
+	if err == nil {
```

Comment:
Do something like this:
```
if err := stop(...); err != nil {
   log.Errorf(".....")
   return
}
deploy(...)
```

This keeps the scope of variables limited.

Reply:
I have made some changes. Can you review them?